src/validation.cairo

```diff
@@ -343,6 +363,50 @@ mod tests {
             txs: ArrayTrait::new().span(),
         };
 
+        let mut txids: Array<Hash> = array![
+            0xacd9825be8bece7782ec746a80b52f44d6a8af41c63dbab59b03e29558469682_u256.into(),
+        ];
+        let merkle_root = merkle_root(ref txids);
+
+        println!("MERKLE: {:?}", merkle_root);
+
+        let block_hash_result = block_hash(@chain_state, @block, merkle_root).unwrap();
+
+        // let block_hash_expected =
+        // 424a5fa890afa8f6b4f06fd7761d643628b8fccc2631de5cb240ca08babaaee5;
+
+        let block_hash_expected = HashTrait::to_hash(
```

Comment:
You can cast block hash to bytearray and use from_hex for the expected value

Reply:
Can u give me an example of that?

Reply:
```assert_eq!(from_hex("1daf02c67eb5e60940caffa259f2bb597639d85d511f03000000000000000000"), block_hash_result.into());
```